akka-stream-typed/src/main/scala/akka/stream/typed/javadsl/ActorSink.scala

```diff
@@ -48,12 +48,19 @@ object ActorSink {
    * will be sent to the destination actor.
    * When the stream is completed with failure - result of `onFailureMessage(throwable)`
    * function will be sent to the destination actor.
+   *
+   * @param ref the receiving actor as `ActorRef<T>` (where `T` must include the control messages below)
+   * @param messageAdapter a function that wraps the stream elements to be sent to the actor together with an `ActorRef[Ack]` which accepts the ack message
+   * @param onInitMessage a function that wraps an `ActorRef<ACK>` into a messages to couple the receiving actor to the sink
+   * @param ackMessage a fixed message that is expected after every element sent to the receiving actor
+   * @param onCompleteMessage the message to be sent to the actor when the stream completes
+   * @param onFailureMessage a function that creates a message to be sent to the actor in case the stream fails from a `Throwable`
    */
-  def actorRefWithBackpressure[T, M, A](
+  def actorRefWithBackpressure[T, M, ACK](
```

Comment:
I don't think I'm a fan of multi-letter names in generics - could we document this in scaladoc (`@tparam`) instead?

Reply:
I wanted to remove the indirection for the brain to keep track of. If you feel strongly about it I'll revert.

There is "prior work", though.
https://github.com/akka/akka/blob/33d64f9dba6b91fbe7c532af178b9677477d0391/akka-stream-typed/src/main/scala/akka/stream/typed/scaladsl/ActorSource.scala#L77

Reply:
But Mima is upset, as well.

Reply:
(mima is too strict here, though - should be fixed in the next release, https://github.com/lightbend/mima/pull/505)

Reply:
I think I may have let the prior art through but I think I agree with @raboof, there's indirection but to me a single letter also highlights that it isn't a type, when I look at full names I kind of have to backtrack from that.